src/View/Helper/LinkHelper.php

```diff
@@ -233,4 +234,44 @@ protected function findFiles(array $filter, string $type) : array
 
         return $files;
     }
+
+    /**
+     * Build object nav
+     *
+     * @param array $data Object nav data
+     * @return string
+     */
+    public function objectNav($data) : string
+    {
+        $prev = '‹';
+        $next = '›';
+        $objectType = Hash::get($data, 'object_type', null);
+        if (!empty($data['prev'])) {
+            $prev = $this->Html->link(
+                $prev,
+                [
+                    '_name' => 'modules:view',
+                    'object_type' => $objectType,
+                    'id' => $data['prev'],
+                    'title' => __('prev'),
```

Comment:
Why add `?title=next` to query string?

Reply:
This is something coming from BEdita3 solution... Let's get rid of it